packages/material/scss/expansion-panel/_theme.scss

```diff
@@ -0,0 +1,12 @@
+@import "~@progress/kendo-theme-default/scss/expansion-panel/_theme.scss";
+
+
+$expander-header-focused-bg: rgba( black, .12 ) !default;
```

Comment:
Move this to the `_variables.scss` file

Reply:
Header focused background should be applied only in the Material theme, therefore I added the variable here. Do you think we should add this variable in the Default and Bootstrap themes too?

Reply:
I am the person who started the initiative to make all of our theme `variables` similar, however, in some situations, I believe this makes no sense.

I do believe that this is one of them. For example, if we make all the `_variables` files identical, and we import the "Default" layout/theme to the other themes - we no longer have a different theme but a swatch.

With the above said, I believe that it is okay to have a different number of variables for the different themes when it makes sense.

We can also add this as a topic for our meeting.

Reply:
Can we have the same variable in the Default and Bootstrap themes, but with `null` value?

Reply:
Yes, we could, but do we need them? We will discuss this case in our meeting